airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -389,9 +389,9 @@ def cast_property_type(self, property_name: str, column_name: str, jinja_column:
             print(f"WARN: Unknown type for column {property_name} at {self.current_json_path()}")
             return column_name
         elif is_array(definition["type"]):
-            return self.cast_property_type_as_array(property_name, column_name)
```

Comment:
why is this changing? isn't this affecting the original BQ destination?

Reply:
This is doing the exact same thing as before (minus the extra function call) so it's not changing anything.

The extra function call was a placeholder to implement the struct/repeated "casting" there but it's actually not doable so not useful anymore